astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -99,14 +99,22 @@ def store_list(option, opt, value, parser):
              'to report per extension (default %default).')
 
     parser.add_option(
-        '-d', '--difference-tolerance', type='float', default=0.,
-        dest='tolerance', metavar='NUMBER',
+        '-rtol', '--relative-tolerance', type='float', default=0.,
+        dest='reltol', metavar='NUMBER',
```

Comment:
Again, this should keep the existing options, but add the new ones as an alternative.  There should be some error if both `-d` and `-rtol` are used and/or conflict.